docs/install_guides/installing-openvino-windows-header.md

```diff
@@ -8,14 +8,20 @@
 
    From Archive <openvino_docs_install_guides_installing_openvino_from_archive_windows>
    From PyPI <openvino_docs_install_guides_installing_openvino_pip>
+   From Anaconda Cloud <openvino_docs_install_guides_installing_openvino_conda>
    Using Docker <openvino_docs_install_guides_installing_openvino_docker_windows>
 
-@endsphinxdirective
 
-If you want to install OpenVINO™ Runtime on Windows, you have the following options: 
+If you want to install OpenVINO™ Runtime on Windows, you have the following options:
+
+* :doc:`Install OpenVINO Runtime from an Archive File <openvino_docs_install_guides_installing_openvino_from_archive_windows>`
+* :doc:`Install OpenVINO from PyPI <openvino_docs_install_guides_installing_openvino_pip>`
+* :doc:`Installing from Anaconda Cloud <openvino_docs_install_guides_installing_openvino_conda>`
```

Comment:
`Installing -> Install` to be consistent with others
maybe `Anaconda Cloud` -> `Conda Forge` ?